wic/verdin-imx8mp-imx-boot-bootpart-2partitions.wks.in

```diff
@@ -0,0 +1,21 @@
+# short-description: Create SD card image with a boot partition
+# long-description:
+# Create an image that can be written onto a SD card using dd for use
+# with i.MX SoC family
+# It uses u-boot + other binaries gathered together on imx-boot file
+#
+# The disk layout used is:
+#  - ---------- -------------- -------------- --------------
+# | | imx-boot |     boot     |    rootfs    |    rootfs2  |
+#  - ---------- -------------- -------------- --------------
+# ^ ^          ^              ^              ^             ^
+# | |          |              |              |             |
+# 0 |        8MiB          72MiB          72MiB            72MiB + rootfs + IMAGE_EXTRA_SPACE (default 10MiB)
```

Comment:
can we actually make a fifth partition called `varfs` or `datafs` or something that takes up `IMAGE_EXTRA_SPACE`